### DISCUSSION TITLE
Customizing the CSS - @tailwindcss/typography

### DISCUSSION THREAD

**Discussion by LukasPolak** [tailwindlabs/tailwindcss#11013] [Help]
In the [Customizing the CSS](https://tailwindcss.com/docs/typography-plugin#customizing-the-css) section, only the `tailwind.config.js` option is mentioned. Can we override typography CSS in the `globals.css` file as well? Is there any drawback?
I like the `globals.css` approach more, as we use tailwind utility classes instead of plain CSS.

Implementation: https://github.com/LukasPolak/loop/blob/1dd0c78d1dda3a2a5441bcd66104caca1def0e3b/src/app/globals.css#L5-L8

**Comment by wongjn** [comment]
Please be aware that your rule differs from `@tailwindcss/typography` behavior in that:

- Your rule will also style `<a>` elements outside elements with the `prose` class name.
- Your rule will also style `<a>` elements inside elements with the `not-prose` class name.

**Comment by LukasPolak** [comment]
Hey @wongjn, thanks for the quick response!
In my case, this behavior does not cause issues, but it is good to keep that in mind.

I will probably refactor the implementation to the config file to mimic the default behavior. 🤔
